src/main/java/com/epam/ta/reportportal/migration/ChangeSets_4_0.java

```diff
@@ -45,13 +46,19 @@
 	@ChangeSet(order = "4.0-1", id = "v.4.0-Update refactored widgets", author = "pbortnik")
 	public void updateWidgets(MongoTemplate mongoTemplate) {
 		final String collection = "widget";
-		Query query = query(where("contentOptions.type").is("line_chart").and("contentOptions.gadgetType").is("old_line_chart"));
+		Criteria criteria = new Criteria();
+		Query query = query(criteria.orOperator(
+				where("contentOptions.type").is("line_chart").and("contentOptions.gadgetType").is("old_line_chart"),
+				where("contentOptions.type").is("trends_chart").and("contentOptions.gadgetType").is("statistic_trend")
+		));
 		query.fields().include("_id");
 		mongoTemplate.stream(query, DBObject.class, collection).forEachRemaining(widget -> {
 			Update update = new Update();
+			String type = (String) ((DBObject) widget.get("contentOptions")).get("type");
 			update.set("contentOptions.type", "trends_chart");
 			update.set("contentOptions.gadgetType", "statistic_trend");
-			update.set("contentOptions.widgetOptions", new BasicDBObject("viewMode", new String[] { "barMode" }));
+			String viewMode = type.equals("line_chart") ? "areaChartMode" : "barMode";
```

Comment:
equals on constant!
do we have constants for the modes?

Reply:
no, we have not